yarn-project/foundation/src/json-rpc/client/json_rpc_client.ts

```diff
@@ -80,7 +80,7 @@ export function makeFetch(retries: number[], noRetry: boolean, log?: DebugLogger
       `JsonRpcClient request ${rpcMethod} to ${host}`,
       makeBackoff(retries),
       log,
-      true,
+      false,
```

Comment:
I had a random json-rpc failure which caused it to retry. It didn't print an error and after the retry I got a 'pop capsule' error :/